packages/ui-drilldown/src/Drilldown/DrilldownOption/props.ts

```diff
@@ -55,12 +55,12 @@ type DrilldownOptionOwnProps = {
    * Label of the Drilldown option.
    */
   children?:
-    | React.ReactNode
-    | ((props: {
-        id: string
-        variant: DrilldownOptionVariant
-        isSelected: boolean
-      }) => React.ReactNode)
+  | React.ReactNode
```

Comment:
I would try to revert these indents if no other code changed :)

Reply:
Reverted, it turned out, I was using a different kind of formatter when saving the files locally instead of prettier.